checks/testdata/workflow-non-github-not-pinned.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 Security Scorecard Authors
```

Comment:
should this file be renamed to `workflow-github-pinned.yaml` since it only contains pinned GitHub-owned actions?